packages/module/patternfly-docs/content/extensions/component-groups/examples/RemoveModal/RemoveModalExample.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import RemoveModal from '@patternfly/react-component-groups/dist/dynamic/RemoveModal';
+import { Button } from '@patternfly/react-core';
+
+export const BasicExample: React.FunctionComponent = () => {
+  const [ isOpen, setIsOpen ] = React.useState(false);
+  return <>
+    <Button onClick={() => setIsOpen(true)}>Open modal</Button>
+    <RemoveModal
+      isOpen={isOpen}
+      title='Remove Item?'
+      confirmButtonLabel='Yup'
```

Comment:
we could make one example with the default button labels and another with custom ones